src/style.css

```diff
@@ -0,0 +1,71 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  box-sizing: border-box;
+}
+
+.form {
+  margin-top: 20px;
+}
+
+.form__label-img {
+  position: absolute;
+  margin-top: 25px;
+  margin-left: 26px;
+  width: 19px;
+  height: 19px;
+}
+
+.small--img {
+  margin-top: 15px;
+  margin-left: 13px;
+  width: 11px;
+  height: 11px;
+}
+
+.form__input {
+  font: inherit;
+  height: 70px;
+  font-size: 16px;
+  text-indent: 61px;
+
+  outline: none;
+  width: 100%;
+  color: #3d4e61;
+  border-radius: 4px;
+  border: 1px solid rgba(225, 231, 237, 1);
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
```

Comment:
Pay attention to CSS properties order
https://9elements.com/css-rule-order/

Reply:
Done 🫡